---
layout:     post
title:      "SCGD 2024"
subtitle:   "From 11EDT onwards, i.e, I missed the first hour's talk (again)"
date:       2024-03-29
author:     "Ruby"
header-img: "img/in-post/post-scg2024/summary.png"
disqus_username: brainfo
catalog: true
catalog_depth: 1
tags:
    - bioinformatics 
    - workshop 
    - single cell technology
---

> all the talks are with preprints

## Genome wide TF binding

### SHARE-Seq

In the aim of interpreting the footprint, in my words, to build a DNA footprint database --> trans-element
<-- DL (inspired by dl)
![from DNA to footprints](/img/in-post/post-scg2024/image.png)
> How stable are the footprint as DNA are stable? \\
> can it output pleipotrip footprints? \\
> the scanning params (padding)

--> use case
identify TFs that drive cell aging --> nucleosomes shift

### 3D spatial from "raw" 2nd seq

![illumina](/img/in-post/post-scg2024/image-1.png)
--> 3D reconstruction based on more raw photonic data
![3D reconstruct](/img/in-post/post-scg2024/image-2.png)
--> expand the *in situ* seq by introducing the expanded hydrogel --> data on microns or DNA distance from lamin

> coupling with lamin to study the chromatin structure is a convention

inactive "X" as an exampled biology to be studied with this
![X](/img/in-post/post-scg2024/image-3.png)

### Sumamry of chromatin resolution and techs

![summary](/img/in-post/post-scg2024/image-4.png)

### Discussion

ATAC replace ChIP
> but it's based on knowledge from ChIP

## open-ST

![summary-openST](/img/in-post/post-scg2024/image-6.png)
![summary-depth-throughput](/img/in-post/post-scg2024/image-7.png)
> 20,000 genes

from Nikolaus Rajewsky lab
![xeniumandvisium](/img/in-post/post-scg2024/image-5.png)

"3D virtual tissue block" \\
module cell segment, module transcripts, map modules

### biological output

3D boundary between cell types, for example, tumor cells and other cells

### Discussion

- sequencing-based has more varied complexed noises. Ideal for hypothesis generating
- imaging-based is ideal for validating the hypothesis.
  
## Rare cell type by na cytometry based PERFF-seq

Coleb Lareau

### study strategy

![strategy](/img/in-post/post-scg2024/strategy.png)

> how much they are differ from other non-rare type?

Criteria for sucessful assay
![criteria](/img/in-post/post-scg2024/rare.png)

Based on

1. RNA FISH
2. 10x Genomics Flex scRNA-seq, where cells are fixed
    > I didn't understand how seq on fixed cells bu two probes on one read

### benchmark using FC/RNA cytometry enrichment

### Biological output

1. enrich cells with TF expression
2. enrich with coding and ncRNA
3. rare cell types enrichment
4. FFEP and frozen tissues

### Discussion

1. easier with RNA vs DNA
2. initiate probes? ~ size of target gene
3. how many markers? limited by cell numbers but not markers

> I would say is the choice of orthogonal markers

## Single cell perturbation dictionaries

1. Chemical (cytokine stimulation)
   - Cui, wt al, Nature (2024)
    > I read this quite carefully already

2. Perturb-Seq
   - first cytokine stimulation
   - then fix and permeablize for perturb-seq (3 guides per gene for 40-60 genes)
  
### Analyzing

1. binarize perturbatoin output as in Nat Genet (2021)
2. imporve to continous score

## spatial and sc by Saturn and star-map

[github](https://github.com/wanglab-broad/spin) \\
[paper](https://www.biorxiv.org/content/10.1101/2023.06.30.547258v1)

### convention

1. smoothing *I missed how*
2. sc analysis
    neighbors are also spatial neighbors

### solution

1. **sample from multiple neighborhoods**
2. smoothing
3. sc analysis

## spatial-resolved sc genomics of human brain by Xiaowei Zhuang

1. space of cells in tissues
2. space of intra-cellular
3. space of chromasome

### Techs

1. MERFISH > 1k genes 2019 Technology

   1. error-robust bc
   2. combinatorial label
   3. sequential imaging

2. MERFISH extened to 3D genomics 2020

localize DNA loci > 1k genomic loci \\
    \+ > 1k nascent RNA \\
    \+ protein
3. extended to epigenomics

with antibody to Tn5 taged protein \\
in a gel so with proximal info \\
--> epigenetic markers
4. thick tissue 3D merfish

200 $\mu m$

### biology

1. Mouse brain cell atlas. related pub: Zhang et al., PMID 38092912 (2023)
somatic contact, paracrine signaling or short-range interactions compose cell-cell interaction
2. Human brain cell atlas

## Discussion

### human mouse diff (behaviral)

- not only from dense (human < 3x mouse)
- modulize effects
- loci to genomic to get chromosome organization by imaging
  - techniqually no problem to increase
  - **high density** in nucleus as a rate limiting --> more runs, so tedious
  - worth efforts
  - now mega base resolution --> higher resolution, you need more runs and sts, imaging time

> I didn't think of a way to get around this density

## Fixed Universal cell embeddings by Yanay Rosen

### Concepts

- e.g. put cells in one universal waddington landscape
- AI e.g. from CZI

### with this, we could model cellular change in silico

Represent cells with
    - representable expression matrix, e.g. low-dimensional embeddings
    - a sentence then can study with NLP
      - feature ordering?
    -  a bag of RNA (no order)
  > I would consider this is prior-free
       -  transformer
Represent genes
    - Excel (redicule)
    - Gene2Vec
    - chatgpt

**Solution:** \\
Gene represent adopting an aa model ESM

**Fixed*:* \\
NOT finetune, so called 0-shot
**Self-supervised**

### therefore

![model](/img/in-post/post-scg2024/model.png)
Output can be massive, and provides a universal space to be mapped
> how high dimension is this bag of RNA representation space

## TIME modality by ZMAN-seq from Ido Amit

![secondseq](/img/in-post/post-scg2024/timeline.png)

![ZMAN-seq overview](/img/in-post/post-scg2024/zman.png)
> basically time stamped

![biology](/img/in-post/post-scg2024/zman_biology.png)

### Makom & Zman-seq

![time-space](/img/in-post/post-scg2024/timespace.png)

## benchmark spatial seq techs

Methods of the Year 2020

### Framework to identify Good data?

![spatial techs](/img/in-post/post-scg2024/spatials.png)

1. targets not overlap across methods
   map to sc and benchmark sc
2. spatial more abundant than sc, why
    1. [x] lack of spcificity: barnyard-like plots to assess specificity
    2. [x] lack of spcificity: coexpression of distinct cell type markers (mutually exclusive)
    3. [x] confound with sensitivity: counts per cell correlate with segmentation size
3. sensitivity and specificity bu controlling for segmentation differences
   > how?
![sens spec](/img/in-post/post-scg2024/sens_spec.png)

### Qang, Huang, et al.,

Xenium, Merscope, CosMx from FFEP

1. sensitivity: by spatial vs sc
2. specificity: %genes above background
   [x] Xenium

### Cook, et al.,

Spatial autocorrelation as a measure of quality. hypothesis: spatial patterning

1. probe detection
2. spatial patterning
3. correlation with sc
4. marker

### open questions

1. tardeoff panel size vs quality
2. segmentation
3. generality
   > if this is necessary?

### Discussion

1. segmentation problem, sts, molecules inaccuratelly assign is one reason, autofluorescence is another among many others
2. recommend barnyard in mouse tissues (staining info from DAPI) is a good approach

## Universal recording of cellular interactions by Sandra Nakandakari-Higa in Mucida Lab

### LIPSTIC based on antigen on immune cells

![lipstic](/img/in-post/post-scg2024/lipstic.png)

### Universal LIPSTIC

![ulipstic](/img/in-post/post-scg2024/ulipstic.png)
> this is good for study syncytiation

[x] quantitative measurement of interaction
[x] outcome of interaction?

### Disccusions

stability, label from 1-3 h depending on tissue and cell type

## My impression

CHANLLENGES make this field worthy devoting
