# About Me

## Basic info

I'm now a master student majored in bioinformatics at the University of Science and Technology of China. I got my graduate degree in Pharmacy( National Base of Biology and Biotechnology) at China Pharmaceutical University under the education of the training plan of national top-notch student in basic sciences. I look forward to be a bioinformatician and to be dedicated in the developmental origins of diseases.

### Academic taste

The first reason that drive me to apply for this PhD position is my academic taste shaped through these years.

I was drown by the topics in embryogenesis, brain sciences, and theoretical biology.  And once in the pharmacy school, we are trained to take the body as a whole while doing experiments to check the specific molecular function or cellular process. I was also a math/physics-minded, wanting mathematically modelling and the creations leveraging the characterization results. For my future years, I'd like to learn the methodologies to uncover the regulatory network underlying diseases by analyze signals measuring the epigenome and transcriptome.

## Research Experience

### piRNA-disease database

Following my tastes, my first contribution is to construct a database encompassing the piRNA-disease relations. since some piRNAs can suppress or promote human diseases by regulating target genes' expression. We collected experimentally supported involvement of piRNAs  in 28 diseases construct a piRNA-disease database.

### HCC CSCs

In an attempt to combining epigenome and transcriptome data to understand disease mechanisms, I found a transcription factor whose motif preferentially accessible and one of its target gene specifically occupies genomic regions in an active compartment, thus find that potential regulatory relation in HCC CSCs.

### Tetrahymena

I also analyzed Hi-C data from the Tetrahymena conjugation process and provide a potential MAC formation hypothesis.

### Spermatogenesis

I also analyze Hi-C and other -omics data sampled from mouse meiotic process and provide evidence testing the functional hypothesis of compartments, TADs and loops.

### TAD clusters

Enlightened by the debates in the TADs function, I started to cluster them based on signals coming from TF binding, histone marks, transcription, chromosomal interaction, adding a layer of the similarities and discrepancies between newly classified TADs.

### Summary of research

The above mentioned research all lack the resolution down to the individual cell level, which is specially important when it comes to individual cells in the embryos with so different fates. All of my previous projects lack the simultaneous measurement of multi-omics, leading to some concerns about the matching of different layers of transcription regulation. In the last TAD clustering research, I don't have enough time to examine each embryonic stages. And some basic science research haven't been linked to the perturbations of the networks and direct linked to the phenotypes.

These limits drive me to push forward the boundary of knowledge in the spatial-time regulation network underlying the embryonic development and to explore the developmental origin of diseases.

## competence

In all, I was trained in basic experiments and I'm eager to learn new methodologies needed to science' sake.

I'm familiar with data processing from the NGS raw data to the interpretation of them using statistics, feature selection or basic machine learning algorithms with shell, python or R scripts.

Being a graduate student, I somewhat got accustomed to regularly reporting and documenting my procedures and results. And I wrote proposals for my supervisors and manuscripts too.

I can get along well with students and colleges from various disciplines. I discuss actively with the engineers at the ICBME conference for example. I worked with students from various department to finish our final projects for classes taught in math, computer sciences, big data department.

These facilitate me fitting the project, leveraging single-cell multi-omics data with bioinformatic specialization to dissect the aberrant developmental progression.
