<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Preprocess | Benchmark snRNA-seq data analysis methods</title>
  <meta name="description" content="This is a minimal benchmark for analyzing snRNA-seq data." />
  <meta name="generator" content="bookdown 0.29 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Preprocess | Benchmark snRNA-seq data analysis methods" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This is a minimal benchmark for analyzing snRNA-seq data." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Preprocess | Benchmark snRNA-seq data analysis methods" />
  
  <meta name="twitter:description" content="This is a minimal benchmark for analyzing snRNA-seq data." />
  

<meta name="author" content="Hong Jiang" />


<meta name="date" content="2022-11-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/viz-1.8.2/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.9/grViz.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> About</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#comments-on-snrna-seq-vs-scrna-seq"><i class="fa fa-check"></i><b>1.1</b> Comments on snRNA-seq vs scRNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="preprocess.html"><a href="preprocess.html"><i class="fa fa-check"></i><b>2</b> Preprocess</a>
<ul>
<li class="chapter" data-level="2.1" data-path="preprocess.html"><a href="preprocess.html#empty-droplets"><i class="fa fa-check"></i><b>2.1</b> Empty droplets</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="preprocess.html"><a href="preprocess.html#dropletutils"><i class="fa fa-check"></i><b>2.1.1</b> DropletUtils</a></li>
<li class="chapter" data-level="2.1.2" data-path="preprocess.html"><a href="preprocess.html#cellbender"><i class="fa fa-check"></i><b>2.1.2</b> Cellbender</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Benchmark snRNA-seq data analysis methods</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="preprocess" class="section level1 hasAnchor" number="2">
<h1><span class="header-section-number">Chapter 2</span> Preprocess<a href="preprocess.html#preprocess" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In this chapter, we deal with empty droplets, ambient RNAs and doublets, especially in snRNA-seq setting.</p>
<div id="empty-droplets" class="section level2 hasAnchor" number="2.1">
<h2><span class="header-section-number">2.1</span> Empty droplets<a href="preprocess.html#empty-droplets" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We have no prior knowledge about whether a particular library (i.e., cell barcode) corresponds to cell-containing or empty droplets. Thus, we need to call cells from empty droplets based on the observed expression profiles. This is not entirely straightforward as empty droplets can contain ambient (i.e., extracellular) RNA that can be captured and sequenced, resulting in non-zero counts for libraries that do not contain any cell.
The aim of empty droplets detection in most cases is to retains nucleus that would have been discarded by other threshold on total counts.</p>
<div id="dropletutils" class="section level3 hasAnchor" number="2.1.1">
<h3><span class="header-section-number">2.1.1</span> DropletUtils<a href="preprocess.html#dropletutils" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://bioconductor.org/packages/release/bioc/html/DropletUtils.html">DropletUtils</a> implemented utils to <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1662-y">distinguish empty droplets</a> and <a href="https://www.nature.com/articles/s41467-018-05083-x">barcode swapping</a>. Here we focused on empty droplet detection:
0. filter out genes with counts of zero for all barcodes
1. estimate the profile of the ambient RNA pool (default as gene count across barcodes with total counts &lt; 100)
2. test each barcode for deviations from this profile using a Dirichlet-multinomial model of UMI count sampling
2.1 Barcodes with significant deviations are considered to be genuine cells
2.2 Use a knee point filter to ensure that barcodes with large total counts are always retained</p>
</div>
<div id="cellbender" class="section level3 hasAnchor" number="2.1.2">
<h3><span class="header-section-number">2.1.2</span> Cellbender<a href="preprocess.html#cellbender" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://cellbender.readthedocs.io/en/latest/">Cellbender</a> provides a deep generative model of background-contaminated counts. The model is used for learning the background RNA profile, distinguishing cell-containing droplets from empty ones, and retrieving background-free gene expression profiles.
The model schema and network representation can be found in <a href="https://www.biorxiv.org/content/10.1101/791699v2.full.pdf">Figure 1g,h</a>
This method has been benchmarked with UMI counts thresholding and DecontX but not DropletUtils
## Ambient contamination</p>
<p>I tested the combinations of the aforementioned empty droplets and ambient RNA detection methods (See diagram) with our in-house data and Iâ€™m convinced that Cellbender and cluster-based methods are of advantage.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/01-Preprocess.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
